src/styles/blocks/stars.css

```diff
@@ -0,0 +1,3 @@
+.stars {
+  display: flex;
+}
```

Comment:
The `.stars` class should include fallback font families to ensure text is rendered with a default system font if the primary font fails to load. Consider adding a generic font family like `sans-serif` as a fallback.